src/components/FormCommentsSort/FormCommentsSort.tsx

```diff
@@ -22,25 +22,79 @@ export interface FormCommentsSortProps {
   council: string;
   reference: string;
   type: "public" | "specialist";
+  defaultContent?: string;
+  defaultResultsPerPage?: number | string | undefined;
 }
 
 export const FormCommentsSort = ({
   defaultOrderBy = "desc",
   council,
   reference,
   type = "public",
+  defaultContent = "",
+  defaultResultsPerPage,
```

Comment:
No need to pass individual props - you can simply pass the entire searchParams object in as `searchParams?: SearchParamsComments`. It keeps the interface tidier and makes the component easier to maintain.